ui/app/services/csp-event.js

```diff
@@ -14,7 +14,7 @@ export default Service.extend({
     return [];
   }),
   connectionViolations: computed('events.@each.violatedDirective', function () {
-    return this.events.filter((e) => e.violatedDirective.startsWith('connect-src'));
+    return this.events.find((e) => e.violatedDirective.startsWith('connect-src'));
```

Comment:
Here is the connectionViolations computer property in a service. Instead of returning an array, I'm returning the first instance. This is used as a boolean check, e.g. does the violation exists or does it not.

Reply:
Weird that's the only place it's used! If that's the case, could we just return true or false since it's used as a boolean check?
```suggestion
    return this.events.some((e) => e.violatedDirective.startsWith('connect-src'));
```

Reply:
Let me try it. It's got some test coverage that helped me figure this out.